pyproject.toml

```diff
@@ -10,31 +10,32 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "3.6.*"
-authutils = "^4.0.0"
+python = "^3.6.1"
+authutils = "^6.0.0"
 cdislogging = "^1.0.0"
-cdiserrors =  "^0.1.2"
+cdiserrors =  "^1.0.0"
 doiclient = {git = "https://github.com/uc-cdis/doiclient", rev = "1.0.0"}
 dosclient = {git = "https://github.com/uc-cdis/dosclient", rev = "1.1.0"}
 gen3authz = "^1.0.4"
```

Comment:
remove file `pkg/gen3authz-1.0.6.tar.gz`

Reply:
ha I can't find this file?

Reply:
it's in your PR files https://github.com/uc-cdis/indexd/pull/311/files 🤷‍♀️

Reply:
it says it's removed though right?when i hover over the red part it says binary file removed

Reply:
ugh sorry, let's just say i'm tired

Reply:
no you're good, I thought it was definitely me because I had it in another branch too ._.